js/scratchx.js

```diff
@@ -177,7 +142,7 @@ function sendURLtoFlash() {
         urls.push(arguments[i]);
     }
     if (urls.length <= 0) return;
-    if (Scratch.FlashApp.ASobj.ASloadGithubURL !== undefined) {
+    if (Scratch.editorIsReady) {
```

Comment:
Testing for `ASloadGithubURL` is risky: if that function is present then the SWF is loaded but it may or may not be ready to receive calls from JS. Instead we now check if the SWF has called `JSeditorReady`, which is a reliable indication that the editor is... ready. Yeah.